### ISSUE TITLE
Time for a new redbean release?

### ISSUE THREAD

**Issue by Dialga** [jart/cosmopolitan#1099]
Just kindly requesting a new release of redbean, considering sqlite version 3.45.1 and lua 5.4.6 has been out.

**Comment by pkulchenko** [comment]
Yes, but neither has been integrated into redbean yet. There is a PR for 5.4.6, but no PR yet for 3.45.1

**Comment by mrdomino** [comment]
What's blocking 5.4.6 being merged?
